devlib/collector/ftrace.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, target,
         self.function_string = None
         self.trace_clock = trace_clock
         self.saved_cmdlines_nr = saved_cmdlines_nr
-        self._reset_needed = True
```

Comment:
This is part of the internal state of the object. Binding it to a tunable will break some flows where tracing is interrupted and restarted.

Reply:
Implemented save/restore logic for kprobes as you recommended.
Thank you!